clients/fides-js/src/fides-tcf.ts

```diff
@@ -175,30 +187,34 @@ async function init(this: FidesGlobal, providedConfig?: FidesConfig) {
   }
   const initialFides = getInitialFides({
     ...config,
-    cookie,
-    savedConsent,
+    cookie: this.cookie,
+    savedConsent: this.saved_consent,
     updateExperienceFromCookieConsent: updateExperienceFromCookieConsentTcf,
   });
   // Initialize the CMP API early so that listeners are established
   initializeTcfCmpApi();
   if (initialFides) {
     Object.assign(this, initialFides);
-    dispatchFidesEvent("FidesInitialized", cookie, config.options.debug);
+    updateWindowFides(this);
+    dispatchFidesEvent("FidesInitialized", this.cookie, config.options.debug, {
+      shouldShowExperience: this.shouldShowExperience(),
```

Comment:
We established below that the `this.experience = ...` assignment on line 203 was needed because otherwise `this.experience` was `undefined` in some cases when calling `await initialize({ ... })` on line 204.

So the corollary: we're *sure* that `this.experience` will be populated by line 200 in the special case when `initialFides` is truthy here, right? Looking at the implementation of `getInitialFides()`, I think that's safe, but... just triple-checking 😉